server/models/Following.js

```diff
@@ -0,0 +1,24 @@
+const mongoose = require("mongoose");
+
+const FollowingSchema = new mongoose.Schema({
+	userId: {
+		type: mongoose.Schema.Types.ObjectId,
+		ref: "User",
+	},
```

Comment:
Sorry If I missed a discussion on this, but is there a reason to have this as a separate model rather than additional following/followers fields on the user model?

Reply:
I created a different model because the task asked me to create a different model for following and followers. Also, I checked with Sina in our today's sprint planning and he was ok with creating another model too. Otherwise, it can be done in the way that you mentioned too. Let me know which way do you think is the best?

Reply:
If Sina said it was fine then it should be fine! :)